bin/manual-testing/wrapped-message-tests.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+var config, Plugin;
+
+config = require("./manual-testing-config");
+Plugin = require("../../lib/index")(config);
+
+
+/**
+ * Instantiates a plugin and uses its connection to clear all records from the
+ * database.
+ *
+ * @param {ddqPlugin~instance} instance
+ * @param {Function} cb
+ */
```

Comment:
Throws were removed based on one of Tyler's other comments, so, for all intents and purposes, fixed.